tests/yamlparser/fixtures/active-choices-param002.yaml

```diff
@@ -1,9 +1,29 @@
 - job:
-    name: unochoice/active-choices-job
+    name: active-choices-job
```

Comment:
I don't get why some jobs are defined in unochoice folder and some others are not. Could you please explain?

Reply:
This is to make sure of name and fullname on XML